security/rbac/docs/index.rst

```diff
@@ -0,0 +1,460 @@
+-  `RBAC documentation <https://docs.confluent.io/current/security/rbac/index.html>`__
+
+
+.. _rbac_demo_local:
+
+Local install
+=============
+
+This method of running the demo is for users who have `downloaded <https://www.confluent.io/download/>`__ |cp| to their local hosts.
+
+Caveats
+-------
+
+-  For simplicity, this demo does not use LDAP, instead it uses the Hash
+   Login service with statically defined users/passwords. Additional
+   configurations would be required if you wanted to augment the demo to
+   connect to your LDAP server.
+-  The |rbac| configurations and role bindings in this demo are not
+   comprehensive, they are only for development to get minimum |rbac|
+   functionality set up across all the services in |cp|.
+   Please refer to the `RBAC
+   documentation <https://docs.confluent.io/current/security/rbac/index.html>`__
+   for comprehensive configuration and production guidance.
+
+Prerequisites
+-------------
+
+1. :ref:`Confluent CLI <cli-install>`:
+   |confluent-cli| must be installed on your machine, version
+   ``v0.127.0`` or higher (note: as of CP 5.3, the |confluent-cli| is a separate
+   :ref:`download <cli-install>`
+2. `Confluent Platform 5.3 <https://www.confluent.io/download/>`__: use
+   a clean install of |cp| without modified properties
+   files, or else the demo is not guaranteed to work
+3. ``jq``
+
+Run the demo
+------------
+
+#. Clone the `confluentinc/examples <https://github.com/confluentinc/examples>`__ repository from GitHub and check out the :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+       git clone git@github.com:confluentinc/examples.git
+       cd examples
+       git checkout |release_post_branch|
+
+#. Navigate to ``security/rbac/scripts`` directory.
+
+   .. codewithvars:: bash
+
+       cd security/rbac/scripts
+
+#. You have two options to run the demo.
+
+-  Option 1: run the demo end-to-end for all services
+
+   .. code:: bash
+
+      $ ./run.sh
+
+-  Option 2: step through it one service at a time
+
+   .. code:: bash
+
+      $ ./init.sh
+      $ ./enable-rbac-broker.sh
+      $ ./enable-rbac-schema-registry.sh
+      $ ./enable-rbac-connect.sh
+      $ ./enable-rbac-rest-proxy.sh
+      $ ./enable-rbac-ksql-server.sh
+      $ ./enable-rbac-control-center.sh
+
+#. After you run the demo, view the configuration files:
+
+   .. code:: bash
+
+      # The original configuration bundled with Confluent Platform
+      $ ls /tmp/original_configs/
+   
+      # Configurations added to each service's properties file
+      $ ls ../delta_configs/
+   
+      # The modified configuration = original + delta
+      $ ls /tmp/rbac_configs/
+
+#. After you run the demo, view the log files for each of the services.
+   Since this demo uses Confluent CLI, all logs are saved in a temporary
+   directory specified by ``confluent local current``.
+
+   .. code:: bash
+
+      $ ls `confluent local current | tail -1`
```

Comment:
remove the `$` and separate the command from the output (so it can be copy/pasted into terminal)

Reply:
Ack